src/Controllers/SuperadminController.php

```
@@ -1352,6 +1352,15 @@ private function packageFormData(): array
             }
         }
 
+        $defaultProvider = $this->aiProviderValue($post['ai_default_provider'] ?? 'google');
+        $defaultModel = $this->aiModelValue(
+            $defaultProvider,
+            $post['ai_default_model'] ?? '',
+            $defaultProvider === 'openai'
+                ? SystemSettings::defaultOpenAiModel()
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\SystemSettings' in method 'packageFormData'. (undefined)

(StaticAccess)